adr/query.py

```diff
@@ -51,6 +52,8 @@ def query_activedata(query):
                              data=query,
                              stream=True)
     response.raise_for_status()
+    if response.json().get('message') is None:
```

Comment:
The `message` key is specific to the `try_usage` query. So this should check if the `data` key itself is empty.

E.g:

    obj = response.json()
    if not obj.get('data'):
        ...
    return obj